app/presenters/expense_presenter.rb

```diff
@@ -76,4 +76,22 @@ def show_map_link?
       true
     end
   end
+
+  def state
+    if expense.mileage_rate_id&.eql?(1) &&
+       expense.calculated_distance.present? &&
+       (expense.distance <= expense.calculated_distance)
+      'Accepted'
+    else
+      'Unverified'
+    end
+  end
+
+  def distance_label
```

Comment:
just naming but i thought this was a CCS class at first - perhaps a `distance_t,`distance_text` or something to reflect it is a translation - 

do you not need to call I18n.t directly?